packages/fbjs/CHANGELOG.md

```diff
@@ -1,20 +1,80 @@
 ## [Unreleased]
 
-### Added
-- More modules, primarily focused on the requirements of Fixed Data Table.
```

Comment:
You shipped a 1.0 with all of this list of things you removed. Some of them may have shipped in the 0.8 releases (though definitely not this one). Please check though.